src/QuickMarcEditor/QuickMarcEditorRows/Leader/Leader.js

```diff
@@ -0,0 +1,60 @@
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import {
+  BytesField,
+  SUBFIELD_TYPES,
+} from '../BytesField';
+import { leaderConfig } from './leaderConfig';
+import { MARC_TYPES } from '../../../common/constants';
+
+const Leader = ({
```

Comment:
Let's call the component and files `LeaderField` to be consisten with other fields